submissions/Roka20012/pop-up-menu/style.css

```diff
@@ -202,3 +158,123 @@
 .more:focus {
   box-shadow: 0px 0px 1px 1px #A8C9FEFF;
 }
+
+.visually-hidden {
+  position: absolute;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
+  height: 0px;
+  width: 0px;
+  margin: -1px;
+  padding: 0;
+  border: 0;
+  top: 50px;
```

Comment:
Properties `top` and `left` needs for positioning default browsers tooltip with an error message. In your case just remove these values.

Reply:
Thank you for noticing) Already done